task/one-shot/src/main.rs

```diff
@@ -8,10 +8,7 @@
 use userlib::*;
 
 #[export_name = "main"]
-fn main() -> ! {
+fn main() {
     sys_log!("Hello world from one-shot task!");
     sys_log!("Exiting now!");
-    kipc::exit_current_task();
```

Comment:
Nice!! I like this :smile: It feels more natural than to add a `kipc::exit_current_task()` and `unreachable!()` in every one-shot task!!